FirebaseAI/Sources/Types/Public/Live/LiveGenerationConfig.swift

```diff
@@ -0,0 +1,152 @@
+  ///     increasing for each repetition. Higher values increase the penalty of repetition,
+  ///     resulting in more diverse output.
+  ///
+  ///     > Note: While both `frequencyPenalty` and `presencePenalty` discourage repetition,
+  ///     > `frequencyPenalty` increases the penalty for *each* repetition of a word/phrase, whereas
+  ///     > `presencePenalty` applies the same penalty regardless of how many times the word/phrase
+  ///     > has already appeared.
+  ///
+  ///     > Important: The range of supported `frequencyPenalty` values depends on the model; see
+  ///     > the
+  ///     > [Cloud documentation](https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/inference#generationconfig)
+  ///     > for more details
+  ///   - responseModalities: The data types (modalities) that may be returned in model responses.
+  ///
+  ///     See the [multimodal
+  ///     responses](https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal-response-generation)
+  ///     documentation for more details.
+  ///
+  ///     > Warning: Specifying response modalities is a **Public Preview** feature, which means
+  ///     > that it is not subject to any SLA or deprecation policy and could change in
+  ///     > backwards-incompatible ways.
+  ///   - speechConfig: Controls the voice of the model, when streaming `audio` via
+  ///     ``ResponseModality``.
+  ///   - inputAudioTranscription: Configures (and enables) input transcriptions when streaming to
+  ///     the model.
+  ///
+  ///     Input transcripts are the model's interpretation of audio data sent to it, and they are
+  ///     populated in model responses via ``LiveServerContent``. When this field is set to `nil`,
+  ///     input transcripts are not populated in model responses.
+  ///   - outputAudioTranscription: Configures (and enables) output transcriptions when streaming to
+  ///     the model.
+  ///
+  ///     Output transcripts are text representations of the audio the model is sending to the
+  ///     client, and they are populated in model responses via ``LiveServerContent``. When this
+  ///     field is set to `nil`, output transcripts are not populated in model responses.
+  ///
+  ///     > Important: Transcripts are independent to the model turn. This means transcripts may
+  ///     > come earlier or later than when the model sends the corresponding audio responses.
+  public init(temperature: Float? = nil, topP: Float? = nil, topK: Int? = nil,
+              candidateCount: Int? = nil, maxOutputTokens: Int? = nil,
+              presencePenalty: Float? = nil, frequencyPenalty: Float? = nil,
+              responseModalities: [ResponseModality]? = nil,
+              speechConfig: SpeechConfig? = nil,
+              inputAudioTranscription: AudioTranscriptionConfig? = nil,
+              outputAudioTranscription: AudioTranscriptionConfig? = nil) {
```

Comment:
I feel we should be consistent with how we name the parameters for these `*Config` types.

Reply:
Yes. Let's do that.